bugwarrior/services/github.py

```diff
@@ -384,6 +399,13 @@ def issues(self):
         log.debug(" Found %i issues.", len(issues))
         issues = list(filter(self.include, issues.values()))
         log.debug(" Pruned down to %i issues.", len(issues))
+        return issues
+
+    def issues(self):
+        if self.query:
+            issues = self._issues_by_query()
+        else:
+            issues = self._issues_by_filters()
```

Comment:
I would think the new `query` option would trigger additional task aggregation rather than act as an alternative to the existing API.